### PR TITLE
preselect default values

### PR DESCRIPTION
**Pull request by michaelmattig**

### CODE REVIEW

**Comment by michaelmattig** [comment]
[[NFDI-Portal II] M.1 Vorauswahl von Daten](https://trello.com/c/J9YE1Z62/997-nfdi-portal-ii-m1-vorauswahl-von-daten)

File: projects/dashboards/nfdi-gdo/src/app/species-selector/species-selector.component.ts [hunk 1]
```diff
@@ -441,7 +441,15 @@ export class SpeciesSelectorComponent implements OnInit, OnDestroy {
...
-        this.dataSelectionService.setTimeSteps([...generateYearlyTimeSteps(START_YEAR, END_YEAR, this.currentMonth)]);
+        this.dataSelectionService.setTimeSteps(
+            [...generateYearlyTimeSteps(START_YEAR, END_YEAR, this.currentMonth)],
+            undefined,
+            new Time('2010-01-01T00:00:00.000Z', '2010-02-01T00:00:00.000Z'),
+        );
+
+        this.selectDragonflySpecies('Anax Imperator');
+        this.selectFishSpecies('Abramis brama');
+        this.selectEnvironmentLayer(this.environmentLayers[0]);
```
Comment (ChristianBeilschmidt) [inline projects/dashboards/nfdi-gdo/src/app/species-selector/species-selector.component.ts:452]: From what I read, there should not be any environmental layer be preselected.

File: projects/dashboards/nfdi-gdo/src/app/species-selector/species-selector.component.ts [hunk 2]
```diff
@@ -441,7 +441,15 @@ export class SpeciesSelectorComponent implements OnInit, OnDestroy {
         });
         this.subscriptions.push(environmentLayerSubscription);
 
-        this.dataSelectionService.setTimeSteps([...generateYearlyTimeSteps(START_YEAR, END_YEAR, this.currentMonth)]);
+        this.dataSelectionService.setTimeSteps(
```
Comment (ChristianBeilschmidt) [inline projects/dashboards/nfdi-gdo/src/app/species-selector/species-selector.component.ts:444]: grafik

I get lots of errors when starting the application.

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: projects/dashboards/nfdi-gdo/src/app/species-selector/species-selector.component.ts (hunk 1 above)
Comment (michaelmattig) [reply projects/dashboards/nfdi-gdo/src/app/species-selector/species-selector.component.ts]: b9bbc1bdac6f0091bf57672ed5eb62c06dbd13ec

File: projects/dashboards/nfdi-gdo/src/app/species-selector/species-selector.component.ts (hunk 2 above)
Comment (michaelmattig) [reply projects/dashboards/nfdi-gdo/src/app/species-selector/species-selector.component.ts:444]: a260571deb701beab012aa92739ec3fcf7963856

Comment (ChristianBeilschmidt) [reply projects/dashboards/nfdi-gdo/src/app/species-selector/species-selector.component.ts:444]: grafik

Unfortunately, I am still getting errors.

**Review by jdroenner — APPROVED** [review]

**Comment by jdroenner** [comment]
bors merge

**Pull request merged**